CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
    strcpy(dest, src); 
  
-   strncpy(dest,src, strlen(dest)); //DevSkim: ignore DS111237,DS154189 
+   strncpy(dest,src, strlen(dest)); 
    
-   strcat(dest, "this is bad");
+   strcat(dest, "this is bad");     //DevSkim: ignore all
```

Comment:
[Banned C function detected (strcat)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS141863.md#ruleDS141863)
If the combination of strings is larger than the destination buffer, strcat will cbuffer overflow the destination buffer

*Fix Guidance:* Use strcat_s or strlcat if possible. strncat's size parameter excludes the null terminator, which leads to frequent mistakes


Suggestions:
1.    strcat_s(dest, <size of dest>,  "this is bad");     //DevSkim: ignore all
2.    strlcat(dest,  "this is bad", <size of dest>);     //DevSkim: ignore all